targets/UnrealMarketplacePlugin/source/PlayFab/Source/PlayFabCommon/Private/PlayFabCommmonUtils.cpp.ejs

```diff
@@ -1,9 +1,27 @@
-﻿<%- copyright %>
+<%- copyright %>
 
 #include "PlayFabCommonUtils.h"
 
 using namespace PlayFabCommon;
 
+// see {your unreal root folder}\Engine\Source\Runtime\Launch\Resources\Version.h for more preprocessor defines
+#if ENGINE_MINOR_VERSION==20
```

Comment:
Do higher version qualify for `==20` or should this be `>=20`?

Reply:
Nvm, I see that your goal was to add logic specifically for 4.20.

Reply:
Nope, this version of GetEnvironmentVariable with 3 parameters is the only way to access the system variables in 4.20. For later versions (21 and 22) there's the single parameter version which is considered the "correct" version (3 params is labeled as deprecated http://api.unrealengine.com/INT/API/Runtime/Core/GenericPlatform/FGenericPlatformMisc/GetEnvironmentVariable/2/index.html)

Reply:
It's actually a fix for less than 4.20.
The current code works for 4.22, but went uncaught until we tried to verify on 4.20.

Reply:
Thanks for the fix